virtual-server/examples/python/main.py

```diff
@@ -84,6 +84,19 @@
 print(vsclient.create(my_virtualserver))
 print(f'VirtualServer status: {vsclient.ready(namespace, name)}')
 
+# Update the manifest and attach directly to Load Balancer
+my_virtualserver['spec']['network']['tcp']['ports'] = []
+my_virtualserver['spec']['network']['udp']['ports'] = []
+my_virtualserver['spec']['network']['directAttachLoadBalancerIP'] = True
+print(vsclient.update(my_virtualserver))
+
+# Restart the Virtual Machine Instance to apply changes.
+print(vsclient.kubevirt_api.stop(namespace, name))
```

Comment:
The Kubevirt client we use does not have a restart command, I use stop/start, and additionally, I can wait until meeting the expected state.

Reply:
We could add a restart function though, right? It would just be a 'restart' sent to the kubevirt_api